src/main/java/com/atlassian/tutorial/api/MyPluginComponent.java

```diff
@@ -0,0 +1,5 @@
+package com.atlassian.tutorial.api;
+
+public interface MyPluginComponent {
```

Comment:
do you need this interface and its implementation?

Reply:
it was created automatically, but i don't use it anywhere